test/extensions/path/uri_template_lib/uri_template_internal_test.cc

```diff
@@ -190,16 +195,15 @@ class ParsePathPatternSyntaxFailure : public testing::TestWithParam<std::string>
 
 INSTANTIATE_TEST_SUITE_P(
     ParsePathPatternSyntaxFailureTestSuite, ParsePathPatternSyntaxFailure,
-    testing::Values("/api/v*/1234", "/api/{version=v*}/1234", "/api/v{versionNum=*}/1234",
```

Comment:
It is very hard to see what has changed here. Was it just `"/api/v{versionNum=*}/1234"` removed?

Reply:
True, this whole block looks awful in the diff. With this change the following are now correct and removed from this test.
`"/api/v{versionNum=*}/1234"` , `"/media/eff456/ll-sd-out.{ext}"` , `"/media/eff456/ll-sd-out.{ext=*}"`, `"/*(/**"`